mongodb_store/scripts/mongodb_play.py

```diff
@@ -120,7 +120,7 @@ def queue_from_db(self, running):
 
         # load message class for this collection, they should all be the same
         msg_cls = mg_util.load_class(documents[0]["_meta"]["stored_class"])
-
+        self.msg_type = documents[0]["_meta"]["stored_type"]
```

Comment:
maybe add a comment saying that msg_type stay the same throughout the TopicPlayer

Reply:
Talked about --- instead of message type check for goal_id attribute